streams/shared/src/main/scala/zio/stream/ZPipeline.scala

```diff
@@ -1810,42 +1813,54 @@ object ZPipeline extends ZPipelinePlatformSpecificConstructors {
    */
   def mapZIOPar[Env, Err, In, Out](n: => Int)(f: In => ZIO[Env, Err, Out])(implicit
     trace: Trace
+  ): ZPipeline[Env, Err, In, Out] =
+    mapZIOPar(n, 16)(f)
+
+  /**
+   * Maps over elements of the stream with the specified effectful function,
+   * executing up to `n` invocations of `f` concurrently. Transformed elements
+   * will be emitted in the original order.
+   *
+   * @note
+   *   This combinator destroys the chunking structure. It's recommended to use
+   *   rechunk afterwards.
+   */
+  def mapZIOPar[Env, Err, In, Out](n: => Int, bufferSize: => Int = 16)(f: In => ZIO[Env, Err, Out])(implicit
```

Comment:
Won't having an overload where the 2nd param has a default argument cause ambiguities when compiling if the method is called as `ZPipeline.mapZIOPar(16)(???)` ?

Reply:
EDIT: I just checked, the method without default argument is prioritised by the compiler automatically